### PR TITLE
SCP-173

### PR DESCRIPTION
**Pull request by Time-Green**
Uses the sprites from that one card

Not very complete, but still menacing. There's no sounds yet and should be AI controlled only

Also almost all code is stolen from /tg/'s statue

### CODE REVIEW

**Comment by Time-Green** [comment]
dont merge yet

**Comment by ike709** [comment]
IMO it'd be way more fun if there was a special role or a ghost role that allowed a player to take control of an SCP, with AI control just being a fallback.

**Comment by JakeDaBoss** [comment]
AI won't control SCP's. If we wanted that, we should've went to make it it's own game.

**Comment by ike709** [comment]
image
🤔

**Comment by HeyBanditoz** [comment]
Screenshots of icons?

**Comment by JakeDaBoss** [comment]
Discussed it with Green. 173 will be an AI SCP until near Alpha, where he will be evaluated for playable antag position.

File: code/modules/SCP/SCPs/SCP-173.dm [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/datum/scp
+	name = "SCP-173"
+	designation = "173"
+	classification = KETER
```
Comment (praisenarsie) [inline code/modules/SCP/SCPs/SCP-173.dm]: it's euclid

File: code/modules/SCP/SCPs/SCP-173.dm [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	melee_damage_upper = 20
+
+	minbodytemp = 0
+
+	faction = list("SCP")
+	move_to_delay = 0 // Very fast
+
+	animate_movement = NO_STEPS
+	see_in_dark = 13
+
```
Comment (praisenarsie) [inline code/modules/SCP/SCPs/SCP-173.dm]: dear god you don't need billions of spacing for vars

**Comment by JakeDaBoss** [comment]
Closed due to moving of repo's.

**Pull request closed without merging**